### DISCUSSION TITLE
[운영체제 1일차] 운영체제 구조와 컴퓨터 구조

### DISCUSSION THREAD

**Discussion by seongeun42** [Shortudy-10th/tech-for-developer#84] [오늘의 주제]
운영체제 구조와 컴퓨터 구조 질문 2인조 1개, 3인조 2

**Comment by zini9188** [comment]
커널에 대해 설명하고, 커널의 역할에 대해 말해주세요

**Comment by zini9188** [comment]
하드웨어에서 인터럽트가 발생하는 경우, 운영체제에서 인터럽트를 처리하는 과정에 대해 설명해주세요.

**Comment by seongeun42** [comment]
컴퓨터의 기본적인 구성 요소에 대해 설명해주세요.

**Comment by watty-exclamationMark** [comment]
운영체제의 중요한 점은 "다중 프로그래밍 (Multi programming)"이 가능하다는 점입니다.
다중 프로그래밍이란 무엇인가요?

**Comment by kjc6735** [comment]
명령어 처리 과정에 대해 설명해주세요.
